exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,3564 @@
+{
```

Comment:
<p>The tables have the data in a normalised form. The views denormalise them. For instance, instead of the topic name you will see a number in the table. The names are collected in a supporting table, named something like XYZ_LOOKUP.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>